dist/angular-patternfly.js

```diff
@@ -1599,6 +1599,159 @@ angular.module('patternfly.charts').directive('pfTrendsChart', function () {
 });
 ;/**
  * @ngdoc directive
+ * @name patternfly.charts.directive:pfUtilizationBarChart
+ *
+ * @description
+ *   Directive for rendering a utilization bar chart
+ *   There are three possible fill colors for Used Percentage, dependent on whether or not there are thresholds:<br/>
+ *   <ul>
+ *   <li>When no thresholds exist, or if the used percentage has not surpassed any thresholds, the indicator is blue.
+ *   <li>When the used percentage has surpassed the warning threshold, but not the error threshold, the indicator is orange.
+ *   <li>When the used percentage has surpassed the error threshold, the indicator is is red.
+ *   </ul>
+ *
+ * @param {object} chartData the data to be shown in the utilization bar chart<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.used   - number representing the amount used
+ * <li>.total  - number representing the total amount
+ * </ul>
+ *
+ * @param {object=} chart-title The title displayed on the left-hand side of the chart
```

Comment:
Same reason as above.  User/developer may want to dynamically change the title.  In general, anything that maybe dynamic should be an scope object rather than an attribute string.